mappings/net/minecraft/command/arguments/BlockStateArgumentParser.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS de net/minecraft/command/arguments/BlockStateArgumentParser
+	FIELD a DISALLOWED_TAG_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
+	FIELD b INVALID_BLOCK_ID_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD c UNKNOWN_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
+	FIELD d DUPLICATE_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
+	FIELD e INVALID_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic3CommandExceptionType;
+	FIELD f EMPTY_PROPERTY_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
+	FIELD g UNCLOSED_PROPERTIES_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
+	FIELD h SUGGEST_DEFAULT Ljava/util/function/Function;
+	FIELD i reader Lcom/mojang/brigadier/StringReader;
+	FIELD j allowTag Z
+	FIELD k blockProperties Ljava/util/Map;
+	FIELD l tagProperties Ljava/util/Map;
+	FIELD m blockId Lqe;
+	FIELD n stateFactory Lbqf;
+	FIELD o blockState Lbqe;
+	FIELD p data Lhs;
+	FIELD q tagId Lqe;
+	FIELD r cursorPos I
+	FIELD s suggestions Ljava/util/function/Function;
+	METHOD a stringifyBlockState (Lbqe;)Ljava/lang/String;
+	METHOD a parsePropertyValue (Lbrh;Ljava/lang/String;I)V
+	METHOD a getSuggestions (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
+	METHOD a suggestPropertyValues (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;Lbrh;)Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;
+	METHOD a suggestTagPropertyValues (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;Ljava/lang/String;)Ljava/util/concurrent/CompletableFuture;
+	METHOD a stringifyProperty (Ljava/lang/StringBuilder;Lbrh;Ljava/lang/Comparable;)V
+	METHOD a parse (Z)Lde;
+	METHOD b getBlockState ()Lbqe;
+	METHOD b suggestBlockPropertiesOrEnd (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
+	METHOD c getNbtData ()Lhs;
+	METHOD c suggestTagPropertiesOrEnd (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
+	METHOD d suggestBlockProperties (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
+	METHOD e parseBlockId ()V
+	METHOD e suggestTagProperties (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
+	METHOD f parseTagId ()V
+	METHOD f suggestMojangson (Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
I'd assume it's a contraction of Mojang and Person. It might be better to make this a little clearer though.

Reply:
It gets used within the vanilla command blocking community to describe NBT serialised to a JSON like string, contracting NBT and JSON rather than Person I'd presume.

Reply:
Yes, Mojangson is a fairly well-known term for Mojang's variation on JSON.